internal/handler/address.go

```diff
@@ -32,40 +32,32 @@ import (
 	"github.com/bitmaelum/bitmaelum-suite/pkg/proofofwork"
 	"github.com/bitmaelum/key-resolver-go/internal/address"
 	"github.com/bitmaelum/key-resolver-go/internal/http"
-	"github.com/bitmaelum/key-resolver-go/internal/organisation"
 	"github.com/bitmaelum/key-resolver-go/internal/reservation"
 )
 
 type addressUploadBody struct {
 	UserHash  hash.Hash                `json:"user_hash"`
 	OrgHash   hash.Hash                `json:"org_hash"`
-	OrgToken  string                   `json:"org_token,omitempty"`
 	PublicKey *bmcrypto.PubKey         `json:"public_key"`
 	RoutingID string                   `json:"routing_id"`
```

Comment:
We should probably add `omitempty` to `RoutingID` aswell

Reply:
good idea..